backend/src/server.ts

```diff
@@ -42,11 +42,13 @@ app.listen({ port: PORT, host: IP }, (err) => {
   // ca.crt is the default CA bundle provided by the service account for kubernetes
   // service-ca.crt is the CA bundle provided by the service account for kubernetes used by prometheus
   // odh-ca-bundle.crt and odh-trusted-ca-bundle.crt are the CA bundles provided by the ODH platform
+  // ca-bundle.crt is the default CA bundle used by the system
   const caPaths = [
     '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt',
     '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt',
     '/etc/pki/tls/certs/odh-ca-bundle.crt',
     '/etc/pki/tls/certs/odh-trusted-ca-bundle.crt',
```

Comment:
where does odh-trusted-ca-bundle.crt stem from?

Reply:
The goal of providing ODH support for the cluster-wide and "self-provided" was to make them available as additional certificates for the application trust bundle.  I would be extremely reluctant to overwrite the default certificate bundle unless ODH was going to 100% own the management & lifecycle of the certificates trust bundle for each component.

Right now the initial compromise is we make both available in hopes that each component can provide full coverage for no self-signed certs (Default), application self-signed certs & proxy w/ self-signed certs.

Reply:
yes, agreed, this is safer, it the sense that the linux default location is not overwritten. It just means a bit more work when i.e. setting default env vars related to CA trust in components, like I allude to for odh-notebook-controller.
I like the approach you took a lot, with having three different bundles used (linux system default location, application DSCI self-signed certs, central cluster config proxy w self-signed certs and private corporate CAs).